projects/goharbor/harbor-chart-server/Makefile

```diff
@@ -0,0 +1,43 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.16"
+
+REPO=chartmuseum
+REPO_OWNER=helm
+REPO_SUBPATH=./cmd/chartmuseum
+
+BASE_IMAGE_NAME?=eks-distro-base
+
+IMAGE_COMPONENT=harbor/harbor-chart-server
+BINARY_TARGET_FILES=harbor-chart-server
+EXTRA_GOBUILD_FLAGS=-mod=vendor
+
+$(call IMAGE_TARGETS_FOR_NAME, harbor-chart-server): harbor-chart-server-useradd/images/export
+
+harbor-chart-server-useradd/images/export: IMAGE_USERADD_USER_NAME=chart IMAGE_USERADD_USER_ID=10000
+
+PRECOMPILE_PREPARE=chartmuseum_prepare
+
+harbor_base_namespace=goharbor
+IMAGE_BUILD_ARGS=harbor_base_namespace
+
+COMPONENT=goharbor/harbor-chart-server
+CLONE_URL=https://github.com/$(REPO_OWNER)/$(REPO).git
```

Comment:
It is to accommodate project structure not aligning with git repo structure so build-tooling can still pull code from right place and generate output to the right place.